jupyterlab_caip_optimizer/src/components/main_area_widget.tsx

```diff
@@ -24,7 +24,7 @@ export const ViewManager = ({ data }: { data: ViewType }) => {
     case 'studyDetails':
       return <StudyDetails studyId={data.studyId} />;
     case 'suggestTrials':
-      return <>Suggest Trials: studyId={data.studyId} </>;
+      return <SuggestTrials studyName={data.studyId} />;
```

Comment:
Minor fix, but could we change the `studyName` props to `studyId` for consistency with other views?

Reply:
Or if `studyName` is more intuitive, we can switch everything to `studyName`